app/views/liquid/_registration_form.html.erb

```diff
@@ -70,7 +70,10 @@
     </tr>
     <tr>
       <td><%= f.label :email %></td>
-      <td><%= f.text_field :email %></td>
+      <td>
+        <%= f.text_field :email %>
+        <span id="email_suggestion"><%= t('registration.did_you_mean', suggestion: '<a id="email_suggestion"></a>').html_safe %></span>
```

Comment:
`id`s are supposed to be unique across a page. Using classes would probably be better.